hieradata/development.yaml

```diff
@@ -142,6 +142,7 @@ govuk::apps::publishing_api::draft_content_store: 'http://draft-content-store.de
 govuk::apps::publishing_api::enable_procfile_worker: false
 govuk::apps::publishing_api::rabbitmq::configure_test_exchange: true
 govuk::apps::publishing_api::suppress_draft_store_502_error: '1'
+govuk::apps::publishing_api::govuk_content_schemas_path: '/var/govuk/govuk-content-schemas'
```

Comment:
Should that be `/var/govuk/govuk-content-schemas/dist` as per "Ensure GOVUK_CONTENT_SCHEMAS_PATH env variable is set and points to the dist directory of govuk-content-schemas"  or is our error wrong?